parts/kubernetesmastervars.t

```diff
@@ -179,7 +178,7 @@
     "agentWindowsSku": "2016-Datacenter-with-Containers",
     "agentWindowsVersion": "2016.0.20170127",
     "singleQuote": "'",
-    "windowsCustomScriptSuffix": " $inputFile = '%SYSTEMDRIVE%\\AzureData\\CustomData.bin' ; $outputFile = '%SYSTEMDRIVE%\\AzureData\\CustomDataSetupScript.ps1' ; Copy-Item $inputFile $outputFile ; Start-Job -command { Invoke-Expression('{0} {1}' -f $outputFile, $arguments) } ; "
```

Comment:
I think I screwed up the base for this commit and then didn't force push after I fixed it. The commit was anthony's commit, which is why the diff looked really weird (and probably why it wasn't working). I just force pushed, let's see if it looks better.